Azure/azure-cli
# [AKS] az aks create and update azure-rbac

**Description**
This PR is to port the support of creating and updating AKS-managed cluster with Azure RBAC from aks-preview extension. It adds a new argument --enable-azure-rbac to az aks create command and --enable-azure-rbac and --disable-azure-rbac to the az aks update command
**Testing Guide**

Create an AKS-managed cluster with Azure RBAC
```
az aks create -g MyResourceGroup -n MyManagedCluster --enable-aad --enable-azure-rbac
```
Update an AKS-managed Azure AD cluster
```
az aks update -g MyResourceGroup -n MyManagedCluster  --enable-azure-rbac
```
```
az aks update -g MyResourceGroup -n MyManagedCluster  --disable-azure-rbac
```
---

This checklist is used to make sure that common guidelines for a pull request are followed.
